tests/test_forgot_password.py

```diff
@@ -40,3 +41,42 @@ def test_forgot_password(app):
     login(page.sign_in_page, USERNAME_PASSW_RECOVERY, OLD_PASSWORD)
     with allure.step('Is user authenticated?'):
         assert page.header.is_logined()
+
+@allure.feature('Forgot password')
+@allure.story('Negative')
+@pytest.mark.skip(reason="just for test skipping")
+def test_forgot_password_skiped(app, make_screen):
+    """ Testcase to renew password using email"""
+    page = app
+    page.header.click_icon()
+    page.header.click_log_in()
+
+    page.sign_in_page.click_forgot_password()
+
+    page.forgot_password_page.enter_login_email(USERNAME_PASSW_RECOVERY)
+    page.forgot_password_page.click_submit_button()
+
+    with allure.step('Is instruction sent on email?'):
+        assert page.vacancies_page.is_instructions_sent()
+    page.vacancies_page.click_ok()
+
+    change_varification_link(USERNAME_PASSW_RECOVERY)
+    link = 'http://localhost:4200/confirmPassword?token=3e83667c-c59c-4fda-aa7a-a47346a3cd6a'
```

Comment:
link in the test? please define it or before the test or move to constants

Reply:
Done